SeaPublicWebsite/Views/EnergyEfficiency/Partials/_DownloadActionPlan.cshtml

```diff
@@ -2,5 +2,8 @@
 
 @using (Html.BeginForm("GenerateActionPlanPdf_Post", "EnergyEfficiency", new { reference = Model.Reference }))
 {
-    <input type="submit" value="Download Action Plan PDF"/>
+    <button type="submit" class="govuk-link sea-download-button @Model.ButtonClass">
+        <img src="/assets/images/download-icon.png" aria-hidden="true" />
+        <span>Download Action Plan PDF</span>
+    </button>
```

Comment:
It could be an actual link though that would require changing the endpoint that creates and returns the Action Plan pdf from a POST request to a GET request. I don't think that's actually a problem as it does GET a thing and doesn't do anything POST-ish, I just wasn't sure what the conventions were here.

There are other buttons pretending to be links on BEIS (e.g. "Cancel and continue without an EPC") but I'm not sure that makes it a good idea - I think I'd prefer it was link-styled rather than button-styled to avoid redesigning things but I'm happy to change it to an actual link instead of a pretend link.